pkg/module/go_get_fetcher_test.go

```diff
@@ -0,0 +1,43 @@
+package module
+
+import (
+	"github.com/spf13/afero"
+	"io/ioutil"
+)
+
+func (s *ModuleSuite) TestNewGoGetFetcher() {
+	r := s.Require()
+	fs := s.fs
```

Comment:
I know it's just a test but no need for the extra variable, you could just call
```go
fetcher, err := NewGoGetFetcher(s.goBinaryName, s.fs, repoURI, version)
```
on the next line.

Reply:
sounds good - will do!